Source/WebCore/Modules/applicationmanifest/ApplicationManifest.h

```diff
@@ -36,6 +36,12 @@
 namespace WebCore {
 
 struct ApplicationManifest {
+    enum class Direction : uint8_t {
+        Auto,
+        LTR, // NOLINT
+        RTL, // NOLINT
```

Comment:
This is fine for now. I think exceptions hard-coded into the linter script for "Ltr" and "Rtl" would be better long-term.

Reply:
I filed PR #32598 to add "LTR" and "RTL" to the allowlist.

Reply:
Cool! We could even fix the Notifications dictionary. 🎉 (as a follow up!)